src/main/java/org/deepsymmetry/beatlink/CdjStatus.java

```diff
@@ -624,6 +638,14 @@ public CdjStatus(DatagramPacket packet) {
         playState3 = findPlayState3();
         firmwareVersion = new String(packetBytes, 124, 4).trim();
         handingMasterToDevice = Util.unsign(packetBytes[MASTER_HAND_OFF]);
+
+        if (Util.isOpusQuad(deviceName)) {
+            trackSourcePlayer = translateOpusPlayerNumbers(packetBytes[40]);
```

Comment:
I’m confused, I can’t seem to find this function?

Reply:
It is in DeviceUpdate. We also need to update the device number.

Reply:
Ah, I forgot about the superclass relationship there. I understood the purpose of the function, but not how it was being found. Thanks!